tests/e2e_logging.py

```diff
@@ -111,13 +116,15 @@ def test_update_lua(network, args):
 
 
 def run(args):
-    hosts = ["localhost", "localhost"]
+    hosts = ["localhost"] * (4 if args.consensus == "pbft" else 2)
```

Comment:
I think we should provide a single value of "f" here, and adjust according to the consensus.